hoth_api.yaml

```diff
@@ -315,6 +315,35 @@ components:
         content:
           type: object
           additionalProperties: true
+    CostRequest:
+      description: >-
+        Represents a request for Cost data to be fulfilled by the ATAT Orchestrator by invoking the appropriate /costs
+        API for a given Cloud Service Provider
+      properties:
+        request_id:
+          description: ID to enable correlating Cost reqeusts with responses
+          type: string
+          format: uuid
+        portfolio_id:
+          type: string
+        target_csp:
+          $ref: "#/components/schemas/TargetCloudServiceProvider"
+        start_date:
+          description: Start date of Cost query
+          type: string
+          format: date
+          example: "2021-07-01"
+        end_date:
+          description: End date of Cost query
+          type: string
+          format: date
+          example: "2022-07-01"
+    CostResponse:
```

Comment:
I would also add `type: object` to the **CostResponse**

Reply:
done